pages/videos/kv[kvid].vue

```diff
@@ -26,13 +27,29 @@
 		}).catch(error => console.error(error));
 	}, { immediate: true });
 
+	// Fetch recommendations
+	watch(() => kvid, () => {
```

Comment:
nit: use async/await. same for other usages of `.then`

```typescript
watch(() => kvid, async () => {
	const api = useApi();
	try {
		const newRecommendations = await api.recommendations(kvid);
		recommmendations.value = newRecommendations;
	} catch (error) {
		console.error(error);
	}
}, { immediate: true });
```

Reply:
.then is a bad practice? I should use await?

Reply:
oh i see, you wrote it the way i should write it